planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -213,8 +220,8 @@ void planning_scene_monitor::PlanningSceneMonitor::initialize(const planning_sce
   publish_planning_scene_frequency_ = 2.0;
   new_scene_update_ = UPDATE_NONE;
 
-  last_update_time_ = ros::Time::now();
-  last_state_update_ = ros::WallTime::now();
+  last_update_time_ = last_robot_motion_time_ = ros::Time::now();
+  wall_last_state_update_ = ros::WallTime::now();
```

Comment:
shouldn't all of these be initialized with `Time(0)`?
At this point we didn't receive anything yet, so I wouldn't claim that we just "updated".